src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../common/variables';
+
+.bcs-ActivityThread {
+    .bcs-ActivityThread-button {
+        margin-left: $sidebarActivityFeedSpacingHorizontal;
+        color: $bdl-box-blue;
+        font-weight: bold;
+
+        &:hover {
+            margin-left: $sidebarActivityFeedSpacingHorizontal;
+            font-weight: bold;
+        }
+    }
+}
+
+.bcs-ActivityThreadReplies {
+    position: relative;
+    padding-left: $sidebarActivityFeedSpacingHorizontal;
+
+    &::before {
+        position: absolute;
+        top: $sidebarActivityFeedSpacingVertical;
+        bottom: $sidebarActivityFeedSpacingVertical;
+        left: $sidebarActivityFeedSpacingHorizontal;
+        width: $bdl-grid-unit;
```

Comment:
Not sure we should use `$bdl-grid-unit` here since that variable is typically used for positioning and padding but in this case we just want the length to be the value of `4px` so maybe create a separate variable for that?